onboarding.md

```diff
@@ -4,13 +4,19 @@ Welcome to Hypha! This page is supposed to help those getting situated in our or
 
 ## Contacting Hypha 
 
-We also have the following communication and contact methods:
+We have the following communication and contact methods:
 
 - 📞 +1-437-887-6936 (We have specific guides for [Accessing][accessing-vm] and [Managing][managing-vm] Voicemail)
 - 📫 #1308-1403 Royal York Road, Etobicoke, Ontario  M9P 0A1
 - 🔗 [hypha.coop](https://hypha.coop)
 - 📧 hello@hypha.coop
 
+## Organizational Structure
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated in Ontario, Canada.
+The business has seven Founding Members, of which three of the four First Directors have signing authority.
```

Comment:
```suggestion
The business has seven Founding Members, of which three of the four First Directors have financial signing authority.
```

Reply:
I can still sign resolutions and all other legal documents, like any other director :)

Reply:
Moved the section and no longer refers to bank signing.

```suggestion
The business has seven Founding Members, of which four Directors have signing authority.
```